packages/ibm-products/src/components/CreateModal/CreateModal.tsx

```diff
@@ -61,7 +61,7 @@ export interface CreateModalProps
   /**
    * The title of the CreateModal is usually the product or service name.
    */
-  title: ReactNode;
+  title: string;
```

Comment:
Wonder why we are changing this to string? normally titles follow a node so users can have more flexibility to add icons etc.